service/controller/v25/resource/tccp/main_stack_test.go

```diff
@@ -1,619 +0,0 @@
-package tccp
-
-import (
-	"context"
-	"fmt"
-	"strings"
-	"testing"
-
-	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
-	"github.com/giantswarm/apiextensions/pkg/clientset/versioned/fake"
-	"github.com/giantswarm/micrologger/microloggertest"
-
-	"github.com/giantswarm/aws-operator/service/controller/v25/controllercontext"
-	"github.com/giantswarm/aws-operator/service/controller/v25/key"
-)
-
-func testConfig() Config {
-	c := Config{}
-
-	c.EncrypterBackend = "kms"
-	c.G8sClient = fake.NewSimpleClientset()
-	c.GuestPrivateSubnetMaskBits = 25
-	c.GuestPublicSubnetMaskBits = 25
-	c.InstallationName = "myinstallation"
-	c.Logger = microloggertest.New()
-
-	return c
-}
-
-func statusWithAllocatedSubnet(cidr string, azs []string) v1alpha1.AWSConfigStatus {
-	var statusAZs []v1alpha1.AWSConfigStatusAWSAvailabilityZone
-	for _, az := range azs {
-		statusAZs = append(statusAZs, v1alpha1.AWSConfigStatusAWSAvailabilityZone{
-			Name: az,
-		})
-	}
-
-	status := v1alpha1.AWSConfigStatus{
-		AWS: v1alpha1.AWSConfigStatusAWS{
-			AvailabilityZones: statusAZs,
-		},
-		Cluster: v1alpha1.StatusCluster{
-			Network: v1alpha1.StatusClusterNetwork{
-				CIDR: cidr,
-			},
-		},
-	}
-
-	return status
-}
-
-func TestMainGuestTemplateGetEmptyBody(t *testing.T) {
-	t.Parallel()
-	customObject := v1alpha1.AWSConfig{}
-
-	c := testConfig()
-	newResource, err := New(c)
-	if err != nil {
-		t.Fatal("expected", nil, "got", err)
-	}
-
-	ctx := context.TODO()
-	cc := controllercontext.Context{}
-	ctx = controllercontext.NewContext(ctx, cc)
-
-	_, err = newResource.getMainGuestTemplateBody(ctx, customObject, StackState{})
-	if err == nil {
-		t.Fatal("expected", nil, "got", err)
-	}
-}
-
-func TestMainGuestTemplateExistingFields(t *testing.T) {
```

Comment:
Dropping the tests as they turned out to be not too useful. I am planning to move the adapters left over to the tccp templates and have rendering tests there.

Reply:
Why are they not useful? I don't quite get this.

Reply:
Their design is wrong. They cause a lot of maintenance. They do not really assure service quality. The rendering will be rewritten anyway so all tests will just become outdated eventually. I want to cleanup all that old crap that stands in our way.